pkg/apis/complianceoperator/v1alpha1/compliancescan_types.go

```diff
@@ -6,21 +6,35 @@ import (
 
 // +genclient
 
+// Represents the status of the compliancescan run.
 type ComplianceScanStatusPhase string
 
 const (
-	PhasePending   ComplianceScanStatusPhase = "PENDING"
+	// PhasePending represents the scan pending to be scheduled
+	PhasePending ComplianceScanStatusPhase = "PENDING"
+	// PhaseLaunching represents being scheduled and launching pods to run the scans
 	PhaseLaunching ComplianceScanStatusPhase = "LAUNCHING"
-	PhaseRunning   ComplianceScanStatusPhase = "RUNNING"
-	PhaseDone      ComplianceScanStatusPhase = "DONE"
+	// PhaseRunning represents the scan being ran by the pods and waiting for the results
+	PhaseRunning ComplianceScanStatusPhase = "RUNNING"
+	// PhaseDone represents the scan pods being done and the results being available
+	PhaseDone ComplianceScanStatusPhase = "DONE"
+)
+
+// Represents the result of the compliance scan
+type ComplianceScanStatusResult string
+
+const (
+	// ResultCompliant represents the compliance scan having succeeded
+	ResultCompliant ComplianceScanStatusResult = "COMPLIANT"
+	// ResultFailure represents a compliance scan pod having failed to run the scan
+	ResultFailure ComplianceScanStatusResult = "FAILURE"
```

Comment:
To indicate that an error occurred during the evaluation, could a different phrase than ``FAILURE`` be used?

The status result of ``FAILURE`` may lead many users/administrators/auditors to believe the system failed the compliance scan.

Reply:
Thanks for the feedback! You're right, FAILURE might be confusing. Would ERROR be a better status?

Reply:
Changed it now. What do you think?

Reply:
@JAORMX yes, ``ERROR`` would be perfect.

Reply:
Hmm, the constant name says `ResultError` but the constant value still says `FAILURE`, did you mean to change both maybe?

Reply:
lol. oops!